include/ultra64/exception.h

```diff
@@ -42,4 +47,15 @@ void __osResetGlobalIntMask(OSHWIntr mask);
 
 extern __osHwInt __osHwIntTable[];
 
+#else
+
+// __osHwInt struct member offsets
+#define HWINTR_CALLBACK 0x00
```

Comment:
I can change the define to `HWINT_CALLBACK` if that makes it any clearer? I'm not sure how much I can do to alleviate the issue though since it's all sdk stuff